ibm/mas_devops/roles/suite_install/defaults/main.yml

```diff
@@ -7,3 +7,10 @@ mas_icr_cp: cp.icr.io/cp
 mas_entitlement_username: cp
 
 mas_icr_cpopen: icr.io/cpopen
+
+ibm_common_services_namespace: ibm-common-services
```

Comment:
Can you add a comment here explaining that this is what controls the install plans that are in scope in the manual approval codepath etc. So that it's obvious to future reader what this is doing here 👍